libs/wingsdk/src/std/struct.ts

```diff
@@ -0,0 +1,31 @@
+import { T1 } from "./generics";
+import { Json } from "./json";
+import { Code, InflightClient } from "../core";
+
+/**
+ * Base class for all structs
+ *
+ * @typeparam T1
+ */
+export class Struct {
```

Comment:
Can we expose the struct's schema through public API: `getSchema`? I think all we need to do is a macro that'll create the validator and returns its internal `v.schemas` field.
I'd the fact that struct's a "schema"''d JSons to be a wing "feature".

Reply:
I was thinking something similar which is why I made _getValidator() its own function rather than creating it in the validate method. Should make supporting other features easier.

Reply:
I think `MyStruct.jsonSchema()` is what the spec says